pages/thc/[processRef]/review.tsx

```diff
@@ -112,12 +163,52 @@ const ReviewPage: NextPage = () => {
     value: "Save and finish process",
   });
 
+  const outsidePropertyImageThumbnails = outsidePropertyImages.result
+    ? outsidePropertyImages.result.images.map((image) => (
+        <Thumbnail
+          key={image}
+          src={image}
+          alt={"Thumbnail of an uploaded image"}
+        />
+      ))
+    : [];
+
   return (
     <MainLayout
       title={PageTitles.Review}
       heading="Review Tenancy and Household Check"
       pausable
     >
+      <SummaryList
+        className="govuk-summary-list--no-border mat-tenancy-summary"
+        rows={[
+          {
+            key: "Address",
+            value: address.result ? address.result.join(", ") : "loading",
+          },
+          {
+            key: "Tenants",
+            value: tenantNames ? tenantNames.join(", ") : "loading",
+          },
+          {
+            key: "Tenure type",
+            value: tenureType.result ? tenureType.result : "loading",
+          },
+          { key: "Ouside Property:", value: outsidePropertyImageThumbnails },
+        ]}
+      />
+      {outsidePropertyImageThumbnails.length > 0 && (
+        <style jsx>{`
+          .mat-tenancy-summary img {
+            margin-right: 2em;
+          }
+        `}</style>
+      )}
+      <Paragraph>
+        The Tenancy and Household Check has now been completed. Please review
+        answers with tenant(s) and ask them to sign.
```

Comment:
```suggestion
        the answers with all present tenants and ask them to sign.
```

I know it differs from the design, but this is clearer, IMO.